packages/swc-angular-plugin/src/testing.rs

```diff
@@ -12,10 +12,10 @@ use swc_ecma_parser::{Syntax, TsConfig};
 /// The typical example is when we add a decorator call in the middle of a function
 /// with no indentation.
 pub fn test_visitor<V>(visitor: V, input: &str, expected: &str) where V: VisitMut {
-    let syntax = Syntax::Typescript(TsConfig::default());
+    let syntax = Syntax::Typescript(TsSyntax::default());
     let transform = as_folder(visitor);
     let actual = Tester::run(|tester| {
-        let result = tester.apply_transform(transform, "input.js", syntax, input)?;
+        let result = tester.apply_transform(transform, "input.js", syntax, Some(true), input)?;
```

Comment:
added new parameter `"is_module"` means is esm module, all typescript components is esm modules when use `import` and `export`